test/test_xla_sharding.py

```diff
@@ -11,38 +11,60 @@
 import torch_xla.utils.utils as xu
 import torch_xla.experimental.xla_sharding as xs
 from torch_xla.experimental.xla_sharded_tensor import XLAShardedTensor
+from torch_xla.experimental.pjrt import using_pjrt
 
 
-@unittest.skipIf((os.getenv('PJRT_DEVICE') == "") or (
-    xm.get_xla_supported_devices("GPU") is not None
-), "PyTorch/XLA SPMD requires PJRT_DEVICE={CPU, TPU}, GPU is currently not supported."
-                )
+@unittest.skipIf(not using_pjrt() or xm.get_xla_supported_devices("GPU"),
+                 f"Requires PJRT_DEVICE set to `TPU` or `CPU`.")
```

Comment:
@will-cromar I think PJRT-GPU single core is ready now?

Reply:
It's blocked from the our SPMD side, once we support TPU, the transition should be easier to GPU -- maybe sometime next year once we are done with the basic/core SPMD features?